### PR TITLE
chore: add 202405 sonic release file

### PR DESCRIPTION
**Pull request by cyw233**
#### Why I did it
On 202405, `release: 'none'` appears in the `/etc/sonic/sonic_version.yml` file

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add a `sonic_release` file to `sonic-buildimage/files/image_config/`, just like what we did before: https://github.com/sonic-net/sonic-buildimage/pull/9985, https://github.com/sonic-net/sonic-buildimage/pull/16364

#### How to verify it
Install the image and run: `cat /etc/sonic/sonic_version.yml`, then verify the following item in the `/etc/sonic/sonic_version.yml` file: `release: '202405'`

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305
- [x] 202405

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: files/image_config/sonic_release [hunk 1]
```diff
@@ -0,0 +1 @@
+202405
```
Comment (yejianquan) [inline files/image_config/sonic_release:1]: @cyw233 please add an empty line on the bottom

**Review by yejianquan — APPROVED** [review]
LGTM

**Comment by yejianquan** [comment]
Hi @bingwang-ms @yxieca , could you please help to merge this PR?

**Review by bingwang-ms — APPROVED** [review]

**Comment by bingwang-ms** [comment]
LGTM.

**Pull request merged**